run-inputs/privbayes-polish.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
This run-input file could be used as an example and to test the full pipeline when writing extra tests